### ISSUE TITLE
Agent gets stuck on reward and two losses become nan

### ISSUE THREAD

**Issue by prstolpe** [ccnmaastricht/dexterous-robot-hand#4]
# Description
It happened twice that the agents reward remained at the same level until forced termination of training. Two of three losses became nan and remain so until forced termination of training. 

# What I've tried

Out of fear that I have interfered with the code in some way, I recloned the repository two times. Since the update on TBPTT, I have not made any contributions to the master branch. Thus I simply merged the master into lstm_experiments and ran tutorial.py. 

After first increasing the reward significantly, the agent falls back onto a stable lower reward for the rest of the training. 

<img width="853" alt="Screenshot 2019-12-15 at 00 10 35" src="https://user-images.githubusercontent.com/54844336/70855752-523d4780-1ed0-11ea-9340-35899151e7d9.png">
## Hyperparameters
<img width="471" alt="Screenshot 2019-12-15 at 00 19 55" src="https://user-images.githubusercontent.com/54844336/70855769-a0524b00-1ed0-11ea-835f-b93201853ecb.png">
My computer does not support a batch_size of 10 and set it to 4. Therefore, I set it to 4 anyways.
